inference/models/yolo_world/yolo_world.py

```diff
@@ -1,17 +1,25 @@
+import os.path
 from time import perf_counter
 from typing import Any, Optional
 
+import clip
```

Comment:
Should we use our own CLIP model implementation from `inference/models/clip`? Some benefits would be it can use the same weights if you have them in cache + utilizes onnxruntime (so would benefit from OpenVINO)

Reply:
Well, we could, but we did not do it in previous iteration and definitely not the fastest move forward.

Reply:
from what I see, the embeddings calculation under onnx would be ~100ms (150ms vs 250ms) faster for 8 classes, so it makes the difference - but there are also limits of batch size to calculate embeddings in our implementation - I could remove it but not sure the implications - as you see - this change requires effort put in investigation.

Reply:
> there are also limits of batch size to calculate embeddings in our implementation

On CPU this shouldn't matter